frontend/src/layout/ErrorBoundary/index.ts

```diff
@@ -0,0 +1 @@
+export { ErrorBoundary } from './ErrorBoundary'
```

Comment:
Having just run across these in https://github.com/PostHog/posthog/pull/8893, I'm not totally sure if we need to make these `index.ts` files. It's just one more place that can go out of sync. There's no real harm in importing from `/ErrorBoundary/ErrorBoundary`.

Reply:
I don't know if this is related, but I was just having build issues after pulling from master. It couldn't build the toolbar + shared dashboard, and it was complaining about import errors in `index.tsx`. I ended up in this PR (because it was the last PR to change that file). I read this comment + made the change locally to remove the `index.ts`. Re-tried the build. It worked. Un did my local change, and it still worked...

I don't know if I was hitting a caching issue, and just touching the file fixed things or if there's something else going on, but I thought I'd share here in case others have a similar issue tomorrow.